contracts/Lottery.sol

```diff
@@ -335,4 +350,44 @@ contract Lottery is Ownable, Initializable {
 
         emit RequestNumbers(lotteryIdCounter_, requestId_);
     }
+
+    function claimWinReward(uint256 _lotteryId, uint256 _ticketId) external {
+        // Checks lottery numbers have not already been drawn
+
+        require(
+            allLotteries_[_lotteryId].lotteryStatus == Status.Completed,
+            "Lottery State incorrect for draw"
```

Comment:
not sure this word correctly represent the require.
suggest -> "Winning Numbers not chosen yet"